hack/testing/entrypoint.sh

```diff
@@ -113,6 +113,13 @@ function run_suite() {
 	os::cmd::expect_success "oc project logging"
 	os::test::junit::declare_suite_end
 
+    local es_pod=$( get_es_pod es )
```

Comment:
im guessing once the uncommented out tests all pass we'll remove these sections?

Reply:
They are passing now - you want me to remove the checking for the indices now?

Reply:
Sure, other than that this seems good to go to me.